### DISCUSSION TITLE
p/73331967c1814df480811eee598e714b/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#38] [Announcements]
# p/73331967c1814df480811eee598e714b/

前言可能许多前端开发者都知道，自从去年 sindresorhus 发表 esm only 的宣言 一年多以来，许多项目开始转向了 esm only，即仅支持 esm 而不支持 cjs，以此来迫使整个生态更快的迁移到 esm only。 一些流行的项目已经这样做了  sindresorhus 维护的上千个 npm 包 node-fetch remark 系列 更多。。。  它们声称：你可以仍然使用现

https://blog.rxliuli.com/p/73331967c1814df480811eee598e714b/

**Comment by deathknight0718** [comment]
这玩意儿简直就是灾难,我现在恨死前端这帮沙雕了.跟TM都逼一样.一个测试框架,自己的bug比谁都多,什么鬼玩意儿.
